recipes/ddocent/version.patch

```diff
@@ -0,0 +1,17 @@
+# fix samtools version check
+# (as is latest dev verson of ddocent which is unfortunatelly not released yet)
+diff -ruN dDocent-2.7.8/dDocent dDocent-2.7.8/dDocent
+--- dDocent-2.7.8/dDocent	2019-05-02 20:51:03.000000000 +0200
++++ dDocent-2.7.8/dDocent	2020-11-03 11:07:17.153106953 +0100
+@@ -28,8 +28,8 @@
```

Comment:
The patch is malformed. You need to either remove the `echo` line or change this one to
```suggestion
@@ -28,8 +28,9 @@
```
(And of course changing it back when the `echo` line is removed again.)